src/components/goalDetail/GoalDoneList.tsx

```
@@ -0,0 +1,26 @@
+import React from 'react';
+import GoalListItem from './GoalListItem';
+
+// Done 타입 정의
+type Done = { id: number; task: string; date: string; done: boolean };
+
+type Props = {
+  list: Done[];
+  onToggle: (id: number) => void;
```

ESLint

[error] 'id' is defined but never used. Allowed unused args must match /^_/u.

(no-unused-vars)